### DISCUSSION TITLE
Darkseer Rework - *normal punch*

### DISCUSSION THREAD

**Discussion by R-janos** [ValveSoftware/Dota2-Gameplay#9301] [Ideas]
I have a suggestion for a darkseer rework.
I find Darkseer's aghanim scepter ability a bit underwhelming, this could be a possible solution.

As my most professional illustration tries to illustrate.
by having the *normal punch* changed to a vector spell, it could be used in combination with his ulti *wall of replica*.

By using it in combination with the ulti *wall of replica* the punch becomes  *BIGGER punch*,  
enemies on the opposite side of the *Wall*  , would take effect of *normal punch*

![Namnlös](https://github.com/ValveSoftware/Dota2-Gameplay/assets/134011790/55cd7a49-5060-462c-8195-9486bacd1f0e)

**Comment by norata35** [comment]
WOWW!? Its possible that way?                                    [My CC Pay](https://www.myccpay.me/)
